docs/source/migration.mdx

```diff
@@ -280,7 +282,18 @@ The Apollo Server core team no longer maintains the following integration packag
 * [`apollo-server-cloudflare`](https://www.npmjs.com/package/apollo-server-cloudflare)
 * [`apollo-server-azure-functions`](https://www.npmjs.com/package/apollo-server-azure-functions)
 
-If you use one of these packages, see [the community integration index](./integrations/integration-index/) to see if an integration exists for your environment yet, or [build your own](./integrations/building-integrations).
+Instead, we now work with the broader community to maintain the following open-source integrations for Apollo Server:
+
+| Web Framework | Integration Package |
```

Comment:
Could you move this into a component shared with integration-index? I hadn't thought of just putting the full table here which does look nice, but would also be nice to not have to update two pages.

Reply:
Oh, brilliant idea! 🧠 Done!